pkgs/top-level/python-packages.nix

```diff
@@ -7482,10 +7482,10 @@ in {
 
   visitor = callPackage ../development/python-modules/visitor { };
 
-  vtk = toPythonModule (pkgs.vtk_7.override {
+  vtk = disabledIf isPy27 (toPythonModule (pkgs.vtk_9.override {
     inherit (self) python;
     enablePython = true;
-  });
+  }));
```

Comment:
seems that `python3Packages.mayavi` is the only package using this, and doesn't support anything but 7. Do you mind postponing this bump?

Reply:
No problem. So I just need to keep the `vtk_9: init at 9.0.1` and remove the other?

Reply:
yea, I would just add `vtk_9` for now

Reply:
once the majority of package consumers switch over, then we can bump the default

Reply:
Done!